Documentation/network/kubernetes/local-redirect-policy.rst

```diff
@@ -53,14 +53,26 @@ the ``localRedirectPolicy`` value to ``true``.
    helm install cilium |CHART_RELEASE| \\
      --set localRedirectPolicy=true
 
-Verify that Cilium agent pod is running.
+
+Rollout the operator and agent pods to make the changes effective:
+
+.. code-block:: shell-session
+
+    $ kubectl rollout restart deploy cilium-operator -n kube-system
+    $ kubectl rollout restart ds cilium -n kube-system
```

Comment:
The step immediately above uses `install`. Why would the user need to restart after installing from scratch with the relevant configuration enabled?

Reply:
Maybe this is a broader docs structure issue where individual pages tend to be written as "install from scratch with X configuration" rather than being structured as "install, then enable this configuration".

With install as the command, I don't think a restart should be necessary. But if the command was `$ helm upgrade cilium |CHART_RELEASE| --namespace kube-system --reuse-values --set ...` then this totally makes sense.

Reply:
:) The motivation behind this change was for the scenarios where users enable the feature on after the fact where cilium was already installed. I'll update the steps to make this more explicit. Thanks!

Reply:
Pushed the changes. PTAL. Thanks!